src/main/java/Duke.java

```diff
@@ -32,19 +34,25 @@ public static void request(){
                 int num = Integer.parseInt(line.substring(5));
                 System.out.println(num);
                 list[num-1].isDone = true;
-                System.out.println("Nice! I've marked this task as done:");
-                System.out.println("[✓] "+ list[num-1].description);
+                System.out.println("____________________________________________________________\n"+
+                        "Nice! I've marked this task as done:");
+                System.out.println("[✓] "+ list[num-1].description + "\n____________________________________________________________\n");
             } else {
                 list[i] = new Task(line);
+                System.out.println("____________________________________________________________");
```

Comment:
You may refactor this as a method to print "______________________" or turn "___________________________" to be a constant